packages/BarChart/src/BarChart.tsx

```diff
@@ -26,9 +30,13 @@ const BarChart: React.FunctionComponent<BarChartProps> = (
 ) => {
   const { dataSet, className, dataTest, ...rest } = props;
 
-  if (!dataSet) {
-    return null;
-  }
+  const [animation, setAnimation] = React.useState(false);
+
+  React.useEffect(() => {
+    setAnimation(!animation);
+    const timer = setTimeout(() => setAnimation(false), 300);
+    return () => clearTimeout(timer);
+  }, []);
```

Comment:
`animation` should be a dependency.

Reply:
```suggestion
  }, [animation]);
```

Reply:
Adding `animation` in dependency will create an infinite loop, the case of figure which I wish here it is to change that value at the time when the component is mounted only.

Reply:
ya the warnings we have for the dependency arrays are annoying :( sometimes you just want it to be empty to only run on mount! but it doesn't like that :(